notebooks/file_calibration.ipynb

```diff
@@ -61,6 +61,7 @@
     "from hera_notebook_templates.data import DATA_PATH as HNBT_DATA\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            #plt.legend(markerscale=20, title=f'{pol}-polarization', loc='lower right')
</pre><p>you can delete this comment, I think</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>

Reply:
Removed